.Lib9c.Tests/Lib9c.Tests.csproj

```diff
@@ -57,5 +57,7 @@
                when the configuration is 'DevEx'.-->
     <ProjectReference Include="..\Lib9c.DevExtensions\Lib9c.DevExtensions.csproj" />
   </ItemGroup>
-
+  <ItemGroup>
+    <Content Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
```

Comment:
Is this change intentional?

Reply:
Amended. https://github.com/greymistcube/lib9c/pull/1/commits/b86894bb798dc07a9668d5e04a5d26b7ec288576